cea/interfaces/dashboard/landing/routes.py

```diff
@@ -46,7 +47,22 @@ def route_project_overview():
 
     # Get the list of scenarios
     scenarios = get_scenarios(project_path)
-    return render_template('project_overview.html', project_name=project_name, scenarios=scenarios)
+
+    # Get scenario descriptions
+    descriptions = {}
+    for scenario in scenarios:
+        descriptions[scenario] = {}
+        scenario_path = os.path.join(project_path, scenario)
+        zone = os.path.join(scenario_path, 'inputs', 'building-geometry', 'zone.shp')
```

Comment:
please don't create paths like this. use the `InputLocator`. everywhere. i'm planning to move the whole locator thing away from using paths in scripts, but DataFrames instead - so we can have a `CeaServerInputLocator` class that can `locator.df.zone` (or similar) and it would work even if the project files are not on the current computer. this would be the case, say, in a distributed context.